### DISCUSSION TITLE
v0.16.17

### DISCUSSION THREAD

**Discussion by sastan** [tw-in-js/twind#290] [General]
- fix: ensure :is(...) selector is probably parsed  99e9f7cb
- fix: fix plugin detection routine (#208)  0468de3d
- chore: Removes unnecessary whitespace from shadow colors (#235)  24c74837
- TypeScript typings improvement: replace `unknown[]` used in `apply()`, `process()`, `tw()`, and `parse()` (#234)  f79c93d2
- feat: Export group expansion utility (#233)  15441649
- docs: Updating quick-reference link (#229)  94dd8e14
- fix: add the missing new color &quot;sky&quot; (#214)  e04ae1f6
- Fix contribution guide dead link in README (#207)  c303c1d6
- Docs: correct typo in variant grouping section (#199)  a1a6dc22
- Update link in tailwind migration guide to point to configuration page (#194)  4a05d6b4
- chore  33ffb719
- test: max-w-... look into width instead on maxWidth in theme in safari  408870cc

https://github.com/tw-in-js/twind/compare/v0.16.16...v0.16.17

<hr /><em>This discussion was created from the release <a href='https://github.com/tw-in-js/twind/releases/tag/v0.16.17'>v0.16.17</a>.</em>

**Comment by rschristian** [comment]
Any chance that this could be published to NPM? Would love to get my hands on #233.

**Reply by sastan** [reply]
Ups... Will do.

**Reply by sastan** [reply]
done

**Reply by rschristian** [reply]
Thanks! I appreciate it.
